pkg/oc/cli/admin/diagnostics/diagnostics/cluster/network/in_pod/util/runtime.go

```diff
@@ -7,8 +7,7 @@ import (
 	"regexp"
 	"strings"
 
-	"github.com/golang/glog"
-
+	kclientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
```

Comment:
For future reference, I'm pretty sure we should be using the public client APIs in `k8s.io/client-go`.